yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -0,0 +1,822 @@
+module vyatta-protocols-frr-isis-v1 {
+  namespace "urn:vyatta.com:mgmt:vyatta-protocols-frr-isis:1";
+	prefix vyatta-protocols-frr-isis-v1;
+
+  import vyatta-protocols-v1 {
+    prefix protocols;
+  }
+  import vyatta-services-v1 {
+    prefix service;
+  }
+  import vyatta-service-snmp-v1 {
+    prefix service-snmp;
+  }
+  import vyatta-types-v1 {
+    prefix types;
+  }
```

Comment:
These imports, and `vyatta-interfaces-bridge-v1` below, can be removed. This will fix a number of warnings from `pyang`:
```
vyatta-protocols-frr-isis-v1.yang:8: warning: imported module vyatta-services-v1 not used
vyatta-protocols-frr-isis-v1.yang:11: warning: imported module vyatta-service-snmp-v1 not used
vyatta-protocols-frr-isis-v1.yang:14: warning: imported module vyatta-types-v1 not used
vyatta-protocols-frr-isis-v1.yang:26: warning: imported module vyatta-interfaces-bridge-v1 not used
```

Reply:
This is Resolved.